src/main/java/io/anserini/rerank/ScoredDocuments.java

```diff
@@ -32,12 +32,17 @@
   public int[] ids;
   // Scores returned from the searcher's similarity
   public float[] scores;
-  
+
+  public TopDocs topDocs;
   public static ScoredDocuments fromTopDocs(TopDocs rs, IndexSearcher searcher) {
+
+
+
```

Comment:
Remove extra empty lines. Here and all throughout.

Reply:
> hi @aalbahem thanks for your contributions! have you run any of the existing regression tests to see if anything breaks?

I did not locally, the only thing that I did was I recorded what RM3 produced using a toy example, It was the same after refactoring. I plan to build a more realistic unit test later on. I was more confident about the code when my PR passed the Travis integration test. This is my first time contributing to an actual open source project and I am more than happy to learn.